experimental/saiexperimentaldasheni.h

```diff
@@ -210,6 +210,24 @@ typedef enum _sai_eni_attr_t
      */
     SAI_ENI_ATTR_V6_METER_POLICY_ID,
 
+    /**
+     * @brief Action set_eni_attrs parameter DASH_TUNNEL_DSCP_MODE
+     *
+     * @type sai_dash_tunnel_dscp_mode_t
+     * @flags CREATE_AND_SET
+     * @default SAI_DASH_TUNNEL_DSCP_MODE_INVALID
+     */
+    SAI_ENI_ATTR_DASH_TUNNEL_DSCP_MODE,
+
+    /**
+     * @brief Action set_eni_attrs parameter DSCP
+     *
+     * @type sai_uint8_t
+     * @flags CREATE_AND_SET
```

Comment:
should we add 
@validonly SAI_ENI_ATTR_DASH_TUNNEL_DSCP_MODE=SAI_DASH_TUNNEL_DSCP_MODE_PIPE_MODEL

Reply:
Done.